File: examples/go/forex/history/getPreviousClose.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/forex"
)

// Returns the data for the previous close
func GetPreviousClose() {
	forexService := forex.NewForexService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := forexService.History(&forex.HistoryRequest{
		Symbol: "GBPUSD",
	})
	fmt.Println(rsp, err)
}

File: examples/js/forex/history/getPreviousClose.js
const { ForexService } = require("m3o/forex");

// Returns the data for the previous close
async function getPreviousClose() {
  let forexService = new ForexService(process.env.MICRO_API_TOKEN);
  let rsp = await forexService.history({
    symbol: "GBPUSD",
  });
  console.log(rsp);
}

getPreviousClose();
